src/ucp/api/ucp.h

```diff
@@ -1607,6 +1676,26 @@ int ucp_request_is_completed(void *request);
 
 /**
  * @ingroup UCP_COMM
+ * @brief Check the status of non-blocking request.
+ *
+ * This routine checks the state of the request and returns its current status.
+ * Any value different from UCS_INPROGRESS means that request is in a completed
+ * state.
+ *
+ * @param [in]  request     Non-blocking request to check.
+ *
+ * @param [out] info        If request is in completed state, it is
```

Comment:
yes - user would expect that for recv request this would not be changed, or event info=NULL could be passed